etl/steps/data/garden/oecd/2024-07-01/road_accidents.meta.yml

```diff
@@ -0,0 +1,120 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Transport
+        - Causes of Death
+    description_from_producer: |-
+      Road accidents are measured in terms of the number of persons injured and deaths due to road accidents, whether
+      immediate or within 30 days of the accident, and excluding suicides involving the use of road motor vehicles. A road motor
+      vehicle is a road vehicle fitted with an engine as the sole means of propulsion and one that is normally used to carry
+      people or goods, or for towing, on the road. This includes buses, coaches, trolleys, tramways (streetcars) and road vehicles
+      used to transport goods and to transport passengers. Road motor vehicles are attributed to the countries where they are
+      registered, while deaths are attributed to the countries in which they occur.
+    display:
+      numDecimalPlaces: 0
+    processing_level: major
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 365
+
+
+tables:
+  road_accidents:
+    variables:
+      accident_deaths:
+        title: Road accident deaths
+        unit: deaths
+        short_unit: deaths
+        description_short: Number of reported deaths due to road accidents.
+        description_processing: |-
```

Comment:
Does this only apply to accident_deaths? If yes, you maybe need to add description_processing for the other indicators (since they are all tagged as with major processing)

Reply:
Will add processing description on indicator level!